contracts/bridge/contracts/binance/LightClientBridge.sol

```diff
@@ -190,7 +193,10 @@ contract LightClientBridge is Bitfield, ValidatorRegistry, GuardRegistry {
     }
 
     function requiredNumberOfValidatorSigs() public view returns (uint256) {
-        return (validatorSetLen * PICK_NUMERATOR) / THRESHOLD_DENOMINATOR;
+        if (validatorSetLen < 36) {
+            return validatorSetLen * 2 / 3 + 1;
+        }
+        return 25;
```

Comment:
25 is still too large for Ethereum, the expect attack cost is `2^k * 𝑚𝑖𝑛𝑠𝑢𝑝𝑝𝑜𝑟𝑡`

if the min support is 4 ether (4 BNB in bsc, $2000), 2^14 is 16384, some value for k between 14 ~ 18 is enough.

If it is for Ethereum, 4 ether is around $16000, k could be smaller.

BTW, could we upgrade this value, since 1 ether's value could be volatile ?

Reply:
Select k = 25, and 4 ether, the expected attach cost is `2^25 * 4 ether = 134,217,728 ether` > `117,445,312.26 ether` which is Ethereum's total supply.
We could change the value a little.